anafi:
  safe_anafi:
    ros__parameters:
      fixed_frame: false  # Fixed frame for velocity commands
      flightplan_file: ~/dev_ws/install/anafi_autonomy/share/anafi_autonomy/missions/test.mavlink  # Absolute path to the FlightPlan file
      followme_mode: 2  # FollowMe mode: 1 = look at the target without moving automatically, 2 = follow the target keeping the same vector, 3 = follow the target keeping the same orientation to its direction, 4 = follow the target as it was held by a leash
      gains:  # Controller gains
        position:  # Position gains
          p: 2.0  # Position proportional gain
          i: 1.0  # Position integral gain
          d: 0.5  # Position derivative gain
          max_i: 0.1  # Position max integral component
        velocity:  # Velocity gains
          p: 9.1  # Velocity proportional gain
          d: 1.3  # Velocity derivative gain
        yaw:  # Yaw gains
          p: 70.0  # Yaw proportional gain
      hand_launch: false  # Enable hand launched takeoff
      landing_control: false  # Enable control during landing
      mission_type: 0  # Mission type: 0 = flight plan, 1 = follow me
      takingoff_control: false  # Enable control during takeoff
      world_frame: false  # Yaw in world frame

